app/routes/dev-sign-in.js

```diff
@@ -125,13 +121,24 @@ export const devLoginHandlers = [
   },
   {
     method: "GET",
-    path: "/dev-defraid",
+    path: devLandingPageUrl,
     options: {
       auth: false,
-      handler: async (request, h) => {
-        setFarmerApplyData(request, "sendBackDevValue", "true");
-        return h.redirect(requestAuthorizationCodeUrl(request, "apply"));
+      handler: async (_request, h) => {
+        return h.view("dev-landing-page");
       },
     },
   },
+  {
+    method: "POST",
```

Comment:
Do we need this separate handler here just to redirect to the GET one above? Can't they be combined into one thing?

Reply:
Done!